dashboard/app/views/plc/enrollment_task_assignments/complete/_script_completion_task.html.haml

```diff
@@ -0,0 +1,9 @@
+Hello, this is a script you have to complete.
+
+- script = @task.script_id ? Script.find(@task.script_id) : nil
```

Comment:
Script_id is a serialized attribute, so it won't automatically pull in script. But I can define script as a method in ScriptCompletion to have the same effect.

All tasks that are rendering _script_completion_task should have an associated script